db/migrate/20181001163831_create_active_admin_comments.rb

```diff
@@ -0,0 +1,17 @@
+class CreateActiveAdminComments < ActiveRecord::Migration::Current
```

Comment:
Thougth this is okay if you want to have active admin coments, its not required in our app to have those

Reply:
Soo. please remove this migration